modules/client-handler/src/testFixtures/java/org/apache/ignite/client/handler/FakeCatalogService.java

```diff
@@ -53,7 +53,7 @@ public FakeCatalogService(int partitions) {
                 0,
                 0,
                 "table",
-                0,
+                invocation.<Integer>getArgument(0) + 10_000,
```

Comment:
please extract `invocation.getArgument(0)` into a variable. Also, why is this change needed?